docs/core/rid-catalog.md

```diff
@@ -106,88 +109,41 @@ See [Prerequisites for .NET Core on Windows](windows-prerequisites.md) for more
 
 ## Linux RIDs
 
-- Portable
-  - `linux-x64`
-- CentOS
-  - `centos-x64`
-  - `centos.7-x64`
-- Debian
-  - `debian-x64`
-  - `debian.8-x64`
-  - `debian.9-x64` (.NET Core 1.1 or later versions)
-- Fedora
-  - `fedora-x64`
-  - `fedora.27-x64`
-  - `fedora.28-x64` (.NET Core 1.1 or later versions)
-- Gentoo (.NET Core 2.0 or later versions)
-  - `gentoo-x64`
-- openSUSE
-  - `opensuse-x64`
-  - `opensuse.42.3-x64`
-- Oracle Linux
-  - `ol-x64`
-  - `ol.7-x64`
-  - `ol.7.0-x64`
-  - `ol.7.1-x64`
-  - `ol.7.2-x64`
-  - `ol.7.3-x64`
-  - `ol.7.4-x64`
+Only common values are listed. For the latest and complete version, see the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo. Devices running a distribution not listed below may work with one of the Portable RIDs. For example, Raspberry Pi devices running a Linux distribution not listed can be targeted with `linux-arm`.
+
+- Portable (.NET Core 2.0 or later versions)
+  - `linux-x64` (Most desktop distributions like CentOS, Debian, Fedora, Ubuntu and derivatives)
+  - `linux-musl-x64` (Lightweight distributions using [musl](https://wiki.musl-libc.org/projects-using-musl.html) like Alpine Linux)
+  - `linux-arm` (Linux distributions running on ARM like Raspberry Pi)
 - Red Hat Enterprise Linux
-  - `rhel-x64`
+  - `rhel-x64` (Alias for `linux-x64` for RHEL above version 6)
```

Comment:
```suggestion
   - `rhel-x64` (Superseded by `linux-x64` for RHEL above version 6)
```

Reply:
The RIDs are never aliases of each other, e.g. 3rd party NuGet packages may still in theory contain binaries specific to `rhel-x64` RID.

Reply:
@jkotas That makes more sense given what I read in the `Runtime IDs` doc linked from the See Also section on how packages are resolved.